#!/usr/bin/env python
PACKAGE = "lab8_localization"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hue_lower", int_t, 0, "Hue Lower", 0, 0, 180)
gen.add("hue_upper", int_t, 0, "Hue Upper", 180, 0, 180)
gen.add("saturation_lower", int_t, 0, "Saturation Lower", 0, 0, 255)
gen.add("saturation_upper", int_t, 0, "Saturation Upper", 255, 0, 255)
gen.add("value_lower", int_t, 0, "Value Lower", 0, 0, 255)
gen.add("value_upper", int_t, 0, "Value Upper", 255, 0, 255)

exit(gen.generate(PACKAGE, "lab8_localization", "Segment"))
